psi4/src/psi4/libfock/jk.h

```diff
@@ -430,12 +439,33 @@ class PSI_API JK {
     *        defaults to false
     */
     virtual void set_do_wK(bool do_wK) { do_wK_ = do_wK; }
+    bool get_do_wK() {return do_wK_;}
+    /**
+    * Set to combine wK integral tensors
+    * @param wcombine do we combine wK matrices?
+    *        defaults to false unless MemDFJK
+    */
+    virtual void set_wcombine(bool wcombine);
+    bool get_wcombine() { return wcombine_; }
+
     /**
     * Set the omega value for wK
     * @param omega range-separation parameter
     */
     void set_omega(double omega) { omega_ = omega; }
 
+    /**
+    * Set the alpha value for w exchange: weight for HF Term                
+    * @param omega_alpha HF-Exchange weight
+    */
+    virtual void set_omega_alpha(double alpha) { omega_alpha_ = alpha; }
+
+    /**
+    * Set the alpha value for w exchange: weight for dampened Term                
+    * @param omega_beta Dampened Exchange weight
+    */
+    virtual void set_omega_beta(double beta) { omega_beta_ = beta; }
```

Comment:
You know more about inheritance and virtual conventions than I do, but should the definitions be removed here since they're irrelevant if wcombine _not_ implemented and insufficient if wcombine _is_ implemented?

should omega_alpha_/beta_ have getters?

Reply:
I'd like to leave the setter in there because I plan on putting wcombine in DiskDFJK. You need at least a dummy definition of the function if you don't want an implementation for every subclass, so I thought it made more sense to put this function in here because it's a choice between `{}` and the implementation in the file.

The answer to the second question is a matter of taste. Some schools of thought say that you should always have getters and setters. Others say that you should only use them if they're called in the code.

I haven't implemented a getter for either because they aren't called.

Reply:
Thanks for the explanation. I guess I like `{}` b/c you never want that code executed (DiskDFJK needs the add'l `dfh_` line like MemDFJK, right?). But ok as-is.

On the second question, good to know the schools of thought. I agree with no getters if it was pure c-side (and if you wanted to access them you'd add getters and recompile anyways). But since there's py-side, it's nice to export them so ppl can probe the object. Up to you.

omega_ and omega_alpha_/beta_ are independent tracks, right? is there any sense in consolidating to omega_alpha_/beta_ only for the member data and preparing for a future of  g/set_omega+omega_alpha or g/set_omega_alpha+omega_beta?

Reply:
I think python side is a good enough reason to add getters. 

omega_alpha_ and omega_beta_ debend on omega_ but not the other way around. Are you suggesting to input a tuple that sets alpha and beta at once or something else?

Reply:
no, I wasn't thinking a tuple, just a setup so that two omega* set commands would work for both DiskDFJK and MemDFJK, rather than three non-independent set commands. but may be worth waiting to see how the multi-omegas pan out.

Reply:
This function definition should be sufficient for DiskDFJK. Would it be ok to just leave this one in there for that?